public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+          description: An object containing embedded resources.
+        balance:
+          description: The guarantee's balance
+          type: number
+      description: A successful balance response.
+    BadRequest:
+      title: BadRequest
+      required:
+        - code
+        - message
+      type: object
+      properties:
+        code:
+          enum:
+            - BAD_REQUEST
+          type: string
+        message:
+          type: string
+          description: The description of the error.
+      description: An error in the supplied payload, or the Guarantee Reference Number did not match the expected format
+    NotFound:
+      title: NotFound
+      required:
+        - code
+        - message
+      type: object
+      properties:
+        code:
+          enum:
+            - NOT_FOUND
+          type: string
+        message:
+          type: string
+          description: The description of the error.
+      description: A guarantee was not found that matched the supplied Guarantee Reference Number and Access Code
+    halLink:
+      title: halLink
+      required:
+        - href
+      type: object
+      properties:
+        href:
+          type: string
+          description: A link to another resource
+      description: A HAL link object
+    Links:
+      title: Links
+      type: object
+      properties:
+        self:
+          allOf:
+            - $ref: '#/components/schemas/halLink'
+            - description: A HAL link object
+      description: An object containing links to other resources
+  parameters:
+    acceptHeader:
+      name: Accept
+      in: header
+      description: >-
+        Specifies the response format and the
+        [version](/api-documentation/docs/reference-guide#versioning) of the API
+        to be used.
+      required: true
+      schema:
+        type: string
+        enum: [
+          "application/vnd.hmrc.2.0+json"
+        ]
+    contentTypeHeader:
+      name: Content-Type
+      in: header
+      description: The media type of the request body.
+      schema:
+        type: string
+        enum:
+          - application/json
+      required: true
+  securitySchemes:
+    userRestricted:
+      type: oauth2
+      description:
+        HMRC supports OAuth 2.0 for authenticating user restricted API requests using an OAuth 2.0 Bearer Token in the AUTHORIZATION header.
+        See https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints for details.
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://api.service.hmrc.gov.uk/oauth/authorize'
+          tokenUrl: 'https://api.service.hmrc.gov.uk/oauth/token'
+          refreshUrl: 'https://api.service.hmrc.gov.uk/oauth/refresh'
+          scopes:
+            common-transit-convention-guarantee-balance: Access all aspects of guarantee balance
+tags:
+  - name: customs
+    description: ''
```

Comment:
Going back to earlier comment about tags, this `tags` snippet might not be needed because it's a small API with only one endpoint (for now).

However, if you want to keep it, I suggest move this snippet above paths and writing the following:
```
tags:
- name: ctc-guarantee-balance-api
  description: >-
    Initiate a guarantee balance request.
```